### PR TITLE
Create claim.notcoin

### PR DESCRIPTION
**Pull request by GoblinsCommunity**
[
    {  
        "subdomain": "claim.notcoin",  
        "url":  "claim.notcoin.is-amaz.ing",
        "email": "me@likhonsheikh.com",
        "type": "CNAME"
    }
]

### CODE REVIEW

**Comment by Joabutt** [comment]
sorry but you cannot have a url point to itself, it must point to external domains

**Comment by Joabutt** [comment]
closing in 5 days if inactive

**Review by Phamlocbull — APPROVED** [review]

File: claim.notcoin [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {  
+        "subdomain": "claim.notcoin",
```
Comment (Phamlocbull) [inline claim.notcoin:3]: Claim not coin you @locpham6699

File: claim.notcoin [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+    {  
+        "subdomain": "claim.notcoin",  
+        "url":  "claim.notcoin.is-amaz.ing",
```
Comment (Phamlocbull) [inline claim.notcoin:4]: image

Id:6893146288

**Review by Phamlocbull — APPROVED** [review]

**Review by Phamlocbull — APPROVED** [review]

**Review by Phamlocbull — COMMENTED** [review]
@phamloc6699

File: claim.notcoin (hunk 1 above)
Comment (Phamlocbull) [inline claim.notcoin:3]: Phamloc6699

File: claim.notcoin (hunk 2 above)
Comment (Phamlocbull) [inline claim.notcoin:4]: 6893146288

File: claim.notcoin [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+        "email": "me@likhonsheikh.com",
```
Comment (Phamlocbull) [inline claim.notcoin:5]: anhsequayveloc@gmail.com

File: claim.notcoin [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+        "type": "CNAME"
```
Comment (Phamlocbull) [inline claim.notcoin:6]: Notcoin

**Review by Phamlocbull — COMMENTED** [review]
@locpham6699
Id :6893146288
X id: @phamloc95982155
Ton waletll: UQC0Oqo1fb5H9zLam4RmUmCFte-fbRU4rlvXNZSA_q-WfofQ

Myton : UQCt-Q2EVMmOEoA5knbu-Y-4YjNen-B4DAUHjr46vbkp8QuL

Thanck you dex

**Pull request closed without merging**